src/main/java/seedu/address/commons/core/Messages.java

```diff
@@ -23,9 +23,12 @@
     public static final String MESSAGE_INVALID_INDEX = "The appointment index provided is not correct";
     public static final String MESSAGE_NOT_PATIENTLIST = "appointment(s) displayed do not belong to the same patient\n";
     public static final String MESSAGE_NOT_STAFFLIST =
-            "duty shift(s) displayed do not belong to the same staff doctor\n";
-    public static final String MESSAGE_NOT_MISSEDLIST = "current displaying appointment are not "
-            + "belong to missed appointment list\n" + "please type command: missedappt";
-    public static final String MESSAGE_NOTHING_SETTLE = "no missed appointment needs to settle";
+            "Duty shift(s) displayed do not belong to the same staff doctor\n";
+    public static final String MESSAGE_NOT_MISSEDLIST =
+            "Only missed appointments can be settled.\n"
+                    + "Plsease first display the missed appointment listing, "
+                    + "using missappt, before using settleappt command.";
+
+    public static final String MESSAGE_NOTHING_SETTLE = "No missed appointment needs to settle";
```

Comment:
Rephrase and move this to your SettleAppCommandParser.java

Reply:
solved